pubsub/gochannel/fanout.go

```diff
@@ -0,0 +1,67 @@
+package gochannel
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/ThreeDotsLabs/watermill"
+	"github.com/ThreeDotsLabs/watermill/message"
+)
+
+type FanOut struct {
```

Comment:
it would be nice to have some godoc here before "undrafting" ;)

Reply:
Added.